### ISSUE TITLE
TranAD Learning

### ISSUE THREAD

**Issue by zamanzadeh** [imperial-qore/TranAD#11]
Hi, 
Based on the article on arXiv the TranAD algorithm supposed to use L1 and L2 for adversarial learning, while in the training implementation of TranAD in "main.py" it seems that only L1 loss function help learning. Would you please clarify usage of L1, L2 and adversarial learning?
Thanks

**Comment by shreshthtuli** [comment]
As noted in the readme, this is a refactored version for ease of use. We are working on creating another branch that includes the final models, loss functions and tuned hyperparameters for all datasets used for result generation.
